### ISSUE TITLE
Update Packages error with Python 2.7.11

### ISSUE THREAD

**Issue by 8u1a** [pypiserver/pypiserver#129]
I receive the following error when running "pypi-server -U" with python 2.7.11:

File "/home/xxxx/pyvirt_2.7.11_pypi/lib/python2.7/site-packages/pypiserver/manage.py", line 48, in make_pypi_client
    return Server(url, transport=transport)
NameError: global name 'Server' is not defined

I believe it's because "Server" is only imported when sys.version_info is 3.0 or greater (line 8 & 9), but it's then used for lower versions in line 48.

**Comment by Oneplus** [comment]
hi @8u1a , I met the same problem with python 2.7.5. Do you have any solution to this? I tried python3.4 with the standalone package, `Server` error is gone but got another exception saying:

```
  File "./pypi-server-standalone.py/pypiserver/manage.py", line 146, in update
TypeError: sequence item 7: expected str instance, tuple found
```

**Comment by 8u1a** [comment]
Hi Oneplus.  I was able to run this with Python 3.5.1.  Are you able to give that a shot?

**Comment by Oneplus** [comment]
Hi @8u1a,  thanks for the tips. Python2.7 support on "update" seems break because https://github.com/Oneplus/pypiserver/blob/master/pypiserver/manage.py#L14 not input rpc server.

Please see pr #136 for my fix.

**Comment by mplanchard** [comment]
I'll be looking into this over the next few days. Regarding #136, it causes test failures for all Python versions, so it won't be accepted as-is. @Oneplus, if you get to this before me, please look into the reasons for test failure & either update tests or the PR.

**Comment by Oneplus** [comment]
Hi @mplanchard , thank you for taking care of this. According to the CI log output, the test process (specifically the `tests/test_server.py`) halt and wait for Password.

![image](https://cloud.githubusercontent.com/assets/745499/15201346/c3efeddc-1822-11e6-85d7-a455bc040781.png)

Same CI problem occurs on the master branch. See https://travis-ci.org/pypiserver/pypiserver. So I think this may not result from my changes but some error in configuration.

**Comment by mplanchard** [comment]
This was resolved by #141 - thanks again for reporting!
